holmes/core/tools_utils/oauth_tool_connector.py

```
@@ -284,18 +284,32 @@ def _log_token_config_mismatch(user_id: str, toolset: Any) -> None:
         """
         try:
             mgr = _get_token_manager()
             oauth_config = toolset._mcp_config.oauth
             cache_key = mgr._get_cache_key(oauth_config, {"user_id": user_id})
             mgr._cache.evict(cache_key)
-            if isinstance(mgr._store, DiskTokenStore):
-                provider_name = oauth_config.authorization_url or "unknown"
-                mgr._store.delete_token(provider_name, user_id=user_id)
+            provider_name = oauth_config.authorization_url or "unknown"
+            mgr._store.delete_token(provider_name, user_id=user_id)
         except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)